modules/trusted-firmware-m/Kconfig.tfm.crypto_modules

```diff
@@ -7,6 +7,13 @@ if BUILD_WITH_TFM
 
 if TFM_PARTITION_CRYPTO
 
+config TFM_CRYPTO_KEY_MODULE_ENABLED
```

Comment:
Doesn't this need some Cmake code to go with it?

Reply:
Yes, I'll fix

Reply:
check now